script/fetchneedles

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 : "${dbuser:="geekotest"}"
-: "${dbgroup:="www"}"
+: "${dbgroup:="nogroup"}"
```

Comment:
I'd say just change it on a worker and see if that works. Then it's fine with me

Reply:
Tested locally in an nspawn container and cloned a job, which worked fine.